src/commands/graph/graph-handler.js

```diff
@@ -66,14 +67,17 @@ const graphHandler = async (args, options, command) => {
   }
 
   if (schema) {
-    generateHandlerByOperationName({
-      generate: codeGenerator.generateHandler,
-      logger: log,
-      netlifyGraphConfig,
-      schema,
-      operationName,
-      handlerOptions,
-    })
+    /* eslint-disable fp/no-loops */
+    for (const operationName of operationNames) {
+      await generateHandlerByOperationName({
```

Comment:
I don't see any async behaviour in `generateHandlerByOperationName`, so do you need to `await` here? And if not, you could use a `forEach` and avoid disabling the ESLint rule?

Reply:
Good catch, @eduardoboucas . All of the `generateHandler` need to be async now (since they're no longer pure functions) - @dwwoelfel is this PR missing some of the upstream changes?

Reply:
@dwwoelfel I'm fine to merge as-is if you want to follow up with the other changes separately.